engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/analyse/FramePointerAnalysis.scala

```diff
@@ -384,8 +480,16 @@ case object FramePointerAnalysis extends IRPass {
   // === Pass Configuration ===================================================
 
   class FramePointerMeta(
```

Comment:
Try to rewrite to:
```
sealed trait Meta {}
case class FramePointerMeta(val framePointer: FramePointer) {}
case class VariableNames(val names: List[String]) {}
```
it will be more readable, understand, standard. Moreover it excludes the case when both `framePointer` and `variableNames` are defined - which is probably right thing to do looking at `AliasAnalysis` metadata structure.

Reply:
Implemented in 1d9be63 and rewritten to Java.